src/routes/authenticationRoute.js

```diff
@@ -112,4 +112,303 @@ authenticationRouter.post('/signup', validateUserSignup, signupAgent);
+ *       "noOfBed": "2",
+ *       "landSize": 200
+ *      }
+ */
+
+
+/**
+ * @swagger
+ * /properties:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *     example: {
+ *         "id": 14,
+ *         "agent_id": 48,
+ *         "image_url": "game.png",
+ *         "title": "duplex",
+ *         "address": "20, Allen str,surulere",
+ *         "city": "Lagos state",
+ *         "land_area": "500",
+ *         "no_of_baths": 3,
+ *         "no_of_beds": 2,
+ *         "land_size": 200
+ *        }
+ */
+  authenticationRouter.get(
+  '/properties', 
+  allProperties
+  );
+
+  /**
+ * @swagger
+ * /property/{id}:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: id
+ *         description: id of property
+ *         in: path
+ *         type: integer
+ *         require: true
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *     example: {
+ *         "id": 14,
+ *         "agent_id": 48,
+ *         "image_url": "game.png",
+ *         "title": "duplex",
+ *         "address": "20, Allen str,surulere",
+ *         "city": "Lagos state",
+ *         "land_area": "500",
+ *         "no_of_baths": 3,
+ *         "no_of_beds": 2,
+ *         "land_size": 200
+ *        }
+ */
+authenticationRouter.get('/property/:id', getPropertyById);
+
+/**
+ * @swagger
+ * /agent/property/{id}:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: id
+ *         description: id of property
+ *         in: path
+ *         type: integer
+ *         require: true
+ *       - name: access_token
+ *         in: header
+ *         type: string
+ *     example: {
+ *       "image": game.png,
+ *       "title": "duplex",
+ *       "address": "20, Allen str,surulere",
+ *       "city": "Lagos state",    
+ *       "landArea": 500,
+ *       "noOfBath": "3",
+ *       "noOfBed": "2",
+ *       "landSize": 200
+ *        }
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *       400:
+ *         description: Bad input supplied
+ *       500:
+ *         description: Internal server error
+ */
+ authenticationRouter.get(
+  '/agent/property/:id', 
+   verifyToken, 
+   agentProperties
+   );
+
+/**
+ * @swagger
+ * /property/{userId}:
+ *   put:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
```

Comment:
Work on your description, why is everything the same?

Reply:
All descriptions have being worked on.